web/Controllers/AdminController.cs

```diff
@@ -74,12 +74,27 @@ public IActionResult CancelAdminAccessRequest()
 
     [Authorize(Roles="Admin")]
     [HttpGet]
-    public async Task<IActionResult> AdminList()
+    [Route("AdminList")]
+    public async Task<IActionResult> GetAdminList()
     {
         AdminListModel adminListModel = new AdminListModel();
+        
         var adminLists = await _adminService.GetAdminLists();
+        
         adminListModel.CurrentAdmins = adminLists.CurrentAdmins;
         adminListModel.PendingAdmins = adminLists.PendingAdmins;
-        return View(adminListModel);
+        
+        return View("AdminList", adminListModel);
+    }
+
+    [Authorize(Roles="Admin")]
+    [HttpPost]
+    public async Task<ActionResult> AcceptAdminRequest(AdminListModel userToAcceptList, int userToAcceptId)
+    {
+        var userToAccept = _adminService.FindUserById(userToAcceptId);
+        
+        _adminService.ApproveAdminAccessRequest(userToAccept);

```

Comment:
Fine either way, but I'd probably combine these two to be honest, just have one function on AdminService that does both.

I slightly dislike leaking database models out of the service layer, since EF will still track their state and then calling `context.SaveChanges()` can do things you don't expect. Anyway that's mostly me being pedantic